src/eigen/diag_lobpcg_hyper.jl

```diff
@@ -1,7 +1,8 @@
 include("lobpcg_hyper_impl.jl")
 
-@timeit timer function lobpcg_hyper(A, X0; maxiter=100, prec=nothing, tol=20size(A, 2)*eps(real(eltype(A))),
-                                    largest=false, n_conv_check=nothing, kwargs...)
+@timer "LOBPCG" function lobpcg_hyper(A, X0; maxiter=100, prec=nothing,
+                                      tol=20size(A, 2)*eps(real(eltype(A))),
+                                      largest=false, n_conv_check=nothing, kwargs...)
```

Comment:
If you move this to the LOBPCG function itself you can again save some boilerplate.